src/audio/google_rtc_audio_processing_mock.c

```diff
@@ -94,3 +161,30 @@ int GoogleRtcAudioProcessingAnalyzeRender_int16(
 		 data, buffer_size);
 	return 0;
 }
+
+void GoogleRtcAudioProcessingParseSofConfigMessage(uint8_t *message,
```

Comment:
this does not appear to be parsing

Reply:
No, the mock does not do the parsing and instead just flags that no parameters were specified.

Reply:
Haha missed I moved into the mock file, thanks for the pointer